spyder/plugins/updatemanager/scripts/install.sh

```diff
@@ -12,8 +12,19 @@ done
 shift $(($OPTIND - 1))
 
 update_spyder(){
-    $conda update -p $prefix -y --file $install_file
-    read -p "Press return to exit..."
+    # Unzip installer file
+    pushd $(dirname $install_file)
+    unzip -o $install_file
```

Comment:
Also, does this `unzip` come from a conda package?

Reply:
`unzip` is available by default on both macOS and Linux.

Reply:
it is not a conda package